diff --git a/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/ScreenStack.kt b/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/ScreenStack.kt
index 75453fa..028ca72 100644
--- a/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/ScreenStack.kt
+++ b/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/ScreenStack.kt
@@ -177,6 +177,7 @@ class ScreenStack(
                                 R.anim.rns_slide_out_to_right,
                             )
                         StackAnimation.SLIDE_FROM_BOTTOM ->
+                            console.log("StackAnimation.SLIDE_FROM_BOTTOM: " + R.anim.rns_slide_in_from_bottom + " " + R.anim.rns_no_animation_medium)
                             it.setCustomAnimations(
                                 R.anim.rns_slide_in_from_bottom,
                                 R.anim.rns_no_animation_medium,
diff --git a/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/ScreenStackFragment.kt b/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/ScreenStackFragment.kt
index b11a19f..5b2d077 100644
--- a/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/ScreenStackFragment.kt
+++ b/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/ScreenStackFragment.kt
@@ -596,6 +596,8 @@ class ScreenStackFragment :
             // and also this is not necessary when going back since the lifecycle methods
             // are correctly dispatched then.
             // We also add fakeAnimation to the set of animations, which sends the progress of animation
+            animation.duration = 50;
+
             val fakeAnimation = ScreensAnimation(fragment).apply { duration = animation.duration }
 
             if (animation is AnimationSet && !fragment.isRemoving) {
