diff --git a/node_modules/react-native-svg-pan-zoom/src/SvgPanZoom.tsx b/node_modules/react-native-svg-pan-zoom/src/SvgPanZoom.tsx
index 8c5b0aa..acdcc91 100644
--- a/node_modules/react-native-svg-pan-zoom/src/SvgPanZoom.tsx
+++ b/node_modules/react-native-svg-pan-zoom/src/SvgPanZoom.tsx
@@ -40,7 +40,8 @@ export interface Props {
   canvasWidth?: number,
   canvasStyle?: ViewStyle,
   viewStyle?: ViewStyle,
-  onZoom?: (zoom: number) => void
+  onZoom?: (zoom: number) => void,
+  onClick: (event: any) => void,
 }
 
 export interface State {
@@ -82,6 +83,7 @@ export default class SvgPanZoom extends Component<Props, State> {
     canvasStyle: {},
     viewStyle: {},
     onZoom: (zoom: number) => {},
+    onClick: (event: any) => {}
   };
 
   mainViewRef: any
@@ -160,9 +162,12 @@ export default class SvgPanZoom extends Component<Props, State> {
         }
 
         if (touches.length === 2) {
+          console.log("zooming internal")
           this.processPinch(touches[0].pageX, touches[0].pageY, touches[1].pageX, touches[1].pageY);
         } else if (touches.length === 1 && !this.state.isScaling) {
+          console.log("touching");
           this.processTouch(gestureState);
+          this.props.onClick(evt)
         }
       },
       onPanResponderTerminationRequest: (evt, gestureState) => true,
